index.html

```diff
@@ -10,4 +10,9 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>  
+  <div class="technoContent">
+  <img class="imgTechno" src="images/react_icon.svg">
+  <p class="text">ReactJS<br><br> <strong>Release:2013</strong><br><br> React is a declarative,<br>component-based <br>Javascript library for<br> building user interfaces.</p> 
```

Comment:
What is this line?! For bold use css `font-weight: bold`. For line break set a width to container.